mantidproject/mantid
# Add peaks in SliceViewer in any coordinate frame

This adds the ability to add peaks to a peaks workspace in the peaks viewer in any coordinate frame.

**To test:**
Here's a script to get you started by creating a MD workspace in Q. The data can be found on Scratch/Sam

```python 
path = '/Users/samueljackson/Desktop/'
peaks = Load(Filename=os.path.join(path, '38868_ruby_peaks.nxs'), OutputWorkspace='38868_ruby_peaks')
ws = Load(Filename=os.path.join(path, '38868_ruby_vana_processed.nxs'), OutputWorkspace='38868_ruby_vana_processed')
md = ConvertToDiffractionMDWorkspace(InputWorkspace=ws, OneEventPerBin=False)
```
 - Right click on `md` and open the slice viewer
 - Drag and drop `peaks` onto the view
 - The little add peaks button should be enabled at the bottom
 - Click the button, then click anywhere in the current slice to add a peak at that location

Fixes #21208 

**Release Notes** 
[See here]()

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
